nixos/modules/services/monitoring/mimir.nix

```diff
@@ -29,7 +29,7 @@ in {
 
   config = mkIf cfg.enable {
     # for mimirtool
-    environment.systemPackages = [ pkgs.mimir ];
+    environment.systemPackages = [ pkgs.grafana-mimir ];
```

Comment:
I think that's because the module was referencing pname directly, see https://github.com/NixOS/nixpkgs/blob/d32a2bf207f41506cbcc86f69e057a47b8a83456/pkgs/servers/monitoring/mimir/default.nix#L3

And in top level is `grafana-mimir` (see pkgs/top-level/all-packages.nix - Line 21795 - Commit cb26845685076ac02be9b9972a712d500fec9384).

Wouldn't it be better to match the package name everywhere?

Reply:
ok I switched the name everywhere to mimir and added an alias for it.
Let me know if I misunderstood you.

Reply:
Yes, I think that having the package with name (pname) and a different alias (top-level) only cause confusion.

Now, nitpick moment (sorry), `grafana-mimir` sounds better to me because of other Grafana projects that we have (`grafana`, `grafana-loki`, `grafana-agent`).

Reply:
grafana-loki was introduced because there is already a loki package.
The only mimir reference is this one, I feel it makes things a little easier to reference.
that being said, if you feel strongly about this, happy to merge an MR that way.